atd-vze/src/views/VZDashboard/VZDashboard.js

```diff
@@ -85,6 +86,52 @@ function VZDashboard() {
           }
         </Col>
       </Row>
+      <Row className='mt-3'>
+        <Col xs="12" sm="6" md="6">
+          <VZLinksWidget
+            header={`Arterial Management Division Overview`}
+            mainText={`Top location overview, by collision types and modes`}
+            icon="fa fa-arrows"
+            color="dark"
+            link="https://app.powerbigov.us/sharedwithme/reports/42c00944-3a44-4d0a-bdd4-d19d7e3647fe/ReportSection512b18e1068b03c18800?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f"
+            target='_bi_amd'
```

Comment:
I was completely unaware of this use of the `target` attribute! I can't believe that I've never thought referencing an already open window instead of creating a new one each time. Very cool to learn.